django/db/backends/postgresql/schema.py

```diff
@@ -21,6 +21,8 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
 
     sql_delete_procedure = 'DROP FUNCTION %(procedure)s(%(param_types)s)'
 
+    sql_unique_constraint = "UNIQUE (%(columns)s) %(defer)s"
```

Comment:
Perhaps this should be `%(deferrable)s` to match `sql_foreign_key_constraint`.

Reply:
I think a different name is better here. While they both work with deferrable constraints, the api is different enough that consistency is probably misleading.